wordpress-mobile/gutenberg-mobile
# Fix for pressing enter on the title creates a <br> tag

Fixes #1078 by instructing AztecWrapper on Android to remove the new-line (converted to BR on toHTML) when pressing enter.

To test:
- Create a new post
- Write a title and press enter
- Switch to HTML 
- See there is no BR tag at the end of the title

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
